rdt-open-reader/src/main/java/io/ona/rdt/util/RDTSyncConfiguration.java

```diff
@@ -28,16 +36,41 @@ public int getSyncMaxRetries() {
 
     @Override
     public SyncFilter getSyncFilterParam() {
-        return BuildConfig.SYNC_FILTER_PARAM.equals(Constants.Config.TEAM)
-                ? SyncFilter.TEAM_ID : SyncFilter.PROVIDER;
+        return BuildConfig.SYNC_FILTER_PARAM.equals(Constants.Config.LOCATION)
```

Comment:
I don't think we should make these changes here since they only apply to a specific variant of the app. The `supervisor` app. 

We should keep the regular apps syncing by team and only apply this to a supervisor app. You can add a different flavour and separate sync configs in the build.gradle for the `supervisor app` variant.

Reply:
Also, convert this section to a switch-case to select the appropriate filter params.